test/common_cuda.py

```diff
@@ -0,0 +1,27 @@
+r"""This file may initialize CUDA context when imported."""
+
+import torch
+import torch.cuda
+
+
+TEST_CUDA = torch.cuda.is_available()
+TEST_MULTIGPU = TEST_CUDA and torch.cuda.device_count() >= 2
+CUDA_DEVICE = TEST_CUDA and torch.device("cuda:0")
+TEST_CUDNN = TEST_CUDA and torch.backends.cudnn.is_acceptable(torch.tensor(1., device=CUDA_DEVICE))
+TEST_CUDNN_VERSION = TEST_CUDNN and torch.backends.cudnn.version()
+
+
+# Used below in `initialize_cuda_context_rng` to ensure that CUDA context and
+# RNG have been initialized.
+__cuda_ctx_rng_initialized = False
```

Comment:
nit: does it make sense to hide this inside initialize_cuda_context_rng()? Something like
```
initialize_cuda_context_rng.__cuda_ctx_rng_initialized = False
```

Reply:
I think that might be a bit confusing. We use this pattern elsewhere too (i.e. in `dataloader.py`).